### ISSUE TITLE
Doesn't work with latest hesim

### ISSUE THREAD

**Issue by vladimir-spintech** [InnovationValueInitiative/IVI-NSCLC#7]
**Steps to reproduce:**
- Follow the [tutorial](https://innovationvalueinitiative.github.io/IVI-NSCLC/articles/tutorial.html)

**Issue:**
Can't construct cost models:
```R
> costmods <- create_costmods(n = n_samples, struct = struct, patients = pats, ae_probs = ae_probs)
Error: There must only be one row for each combination of sample, strategy_id, state_id, and time_start in 'tbl'.
```

Appears to be broken starting from this commit: https://github.com/hesim-dev/hesim/commit/fdc41c0223cfa17b00c9b103b5834b94c5b0885c

**Comment by dincerti** [comment]
Yeah, this is as expected. A minor update will be needed given the change in hesim. 

But it should still work with the hesim version on CRAN. I'd wait to do any updates until the development version of hesim moves to CRAN.

**Comment by munenobuk** [comment]
Still, the same error happening.

**Comment by dincerti** [comment]
> Still, the same error happening.

Yes, the `IVI-NSCLC` package will need to be updated to reflect changes in `hesim`.

**Comment by munenobuk** [comment]
What modifications are needed to make it usable?

**Comment by munenobuk** [comment]
txseq_dt doesn’t fit to the tbl in the stateval_tbl, since rows are not seen as unique.
I cannot find any more clues and what to do.

**Comment by helendakin** [comment]
I wondered if there is yet a solution that will allow the IVI-NSCLC model to run on the latest version of hesim. I am getting the error "'cost_models' must be a list of objects of class 'StateVals'". 

Does anyone have any script or commands that will allow the model to run?
